src/txdb.cpp

```diff
@@ -168,9 +168,10 @@ bool CBlockTreeDB::ReadLastBlockFile(int &nFile) {
     return Read(DB_LAST_BLOCK, nFile);
 }
 
-CCoinsViewCursor *CCoinsViewDB::Cursor() const
+std::unique_ptr<CCoinsViewCursor> CCoinsViewDB::Cursor() const
 {
-    CCoinsViewDBCursor *i = new CCoinsViewDBCursor(const_cast<CDBWrapper&>(*m_db).NewIterator(), GetBestBlock());
+    std::unique_ptr<CCoinsViewDBCursor> i{
+        new CCoinsViewDBCursor(const_cast<CDBWrapper&>(*m_db).NewIterator(), GetBestBlock())};
```

Comment:
```suggestion
    auto i{std::make_unique<CCoinsViewDBCursor>(const_cast<CDBWrapper&>(*m_db).NewIterator(), GetBestBlock())};
```

Can be written shorter with make_unique?

Reply:
Good call!